_source/_posts/2022-09-08-react-fastify-postgres.md

```diff
@@ -0,0 +1,917 @@
+---
+layout: blog_post
+title: ""
+author: gabi-dombrowski
+by: contractor
+communities: [javascript]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+In this tutorial, we'll set up an app to help us keep track of what NASA facilities we've visited and which ones we still want to visit.
+
+Our app will be a monorepo with Okta authentication using React for the frontend and Fastify for the backend. [Fastify](https://www.fastify.io/) is a highly performant web framework with low overhead that we'll connect to a PostgreSQL database. We'll also use [Lerna](https://lerna.js.org/) to manage the frontend and backend apps in a monorepo.
+
+**Prerequisites**
+
+*Create React App* currently requires Node >= 14.0.0 and npm >= 5.6. The latest required versions can be found at [https://reactjs.org/docs/create-a-new-react-app.html](https://reactjs.org/docs/create-a-new-react-app.html).
```

Comment:
What version of npm were you on? `18.12.1` & `19.0.1` don't seem to have an issue. Those are the LTS and current release versions.

Reply:
I'm running npm v 8.15, but my node version is 16.

We need to explicitly state which versions of Node and npm were used then. Because my version matches the criteria required for the `create-react-app` as documented in the post.

Reply:
I added this line, let me know if it doesn't suffice:

`This tutorial was created using Node v18.8.1 and npm v8.11.0.`